src/main/java/bio/terra/cli/command/resource/update/BqDataset.java

```diff
@@ -17,6 +20,7 @@
     showDefaultValues = true)
 public class BqDataset extends BaseCommand {
   @CommandLine.Mixin ResourceUpdate resourceUpdateOptions;
+  @CommandLine.Mixin BqDatasetLifetime bqDatasetLifetime;
```

Comment:
nit: same as above, can you make the variable name consistent with the above using the `Options` suffix?